GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLUploader.m

```diff
@@ -31,14 +31,15 @@
 
 #import "GDTCCTLibrary/Protogen/nanopb/cct.nanopb.h"
 
+#if DEBUG
```

Comment:
Generally, I prefer if !NDEBUG, since DEBUG isn't part of any standard.

Reply:
Changed, thanks.